gallery/package.json

```diff
@@ -7,7 +7,7 @@
     "start": "npm-run-all lint webpack-dev-server",
     "build": "npm-run-all lint webpack copy-assets",
     "copy-assets": "cpx \"{src/index.html,src/assets/images/favicon*}\" dist",
-    "lint": "eslint 'src/**/*.{js,ts}{,x}'",
+    "lint": "eslint \"./src/**/*.{js,ts}{,x}\"",
```

Comment:
I don't think this'll do what we want on *nix.  Or at least it won't do what it used to.  With the old single quotes the glob string was being passed as-is to `eslint`, and interpreted there.  With double quotes, the shell will expand the glob and pass the results to eslint as separate parameters.  I'd rather not be in that situation due to differences in glob support on different shells (bash doesnt' support `**` for instance).  Is there another way to get it working on windows that doesn't involve double quotes?

Reply:
> I don't think this'll do what we want on *nix. Or at least it won't do what it used to. With the old single quotes the glob string was being passed as-is to eslint, and interpreted there. With double quotes, the shell will expand the glob and pass the results to eslint as separate parameters. I'd rather not be in that situation due to differences in glob support on different shells (bash doesn't support ** for instance (edit: apparently it does as of version 4)). Is there another way to get it working on windows that doesn't involve double quotes?

As I dig into this more, I see we have some other double quoted arguments already that might have the same concern.  Doing some research...

Reply:
> As I dig into this more, I see we have some other double quoted arguments already that might have the same concern. Doing some research...

Oh I was wrong about nix shells expanding globs in double quotes.  It seems that does not in fact happen, so we're good here.

Reply:
see, just wait long enough and the problems just vanish into thin air :D 

seriously though, thanks for the insight @rpokorny (and the ultimate validation that we should be ok).  I'm assuming that CI runs on the branch would show any potential *nix issues, is that a reasonable assumption ?

Reply:
> I'm assuming that CI execution of the branch would show any potential *nix issues, is that a reasonable assumption ?

Well kinda, what I had been thinking was that this would still work just in a sketchy way (the glob being processed by the shell not eslint) that might be OS-dependent.  But again, I think it's fine.